index.php

```diff
@@ -6,7 +6,9 @@
 use GameListing\ViewHelpers\GameViewHelper;
 $db = new PDO('mysql:host=db;dbname=games', 'root', 'password');
 $games = GamesHydrator::getAllGames($db);
-
+if(isset($_GET['query'])){
+    $gamesByQuery = GamesHydrator::getGamesByTitle($db, $_GET['query']);
```

Comment:
this line should be changed as per my other comment

Reply:
Also done